### PR TITLE
CSHARP-4921: Improve Nuget packages hygiene

### PR DESCRIPTION
**Pull request by BorisDog**
- Removed contributors section
- Added release notes link
- Added readme
- Updated icon

### CODE REVIEW

**Comment by rishitb-mongodb** [comment]
@BorisDog Can we use this image instead? Its the same branding logo with the suggested darker background based on our branding guidelines. It should comply with the NuSpec guidelines too but let me know if it doesn't work. 
MongoDB_NuSpecIcon

**Comment by rishitb-mongodb** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -76,96 +76,4 @@ Contributing
...
-* Craig Wilson              https://github.com/craiggwilson
-* Zhmayev Yaroslav          https://github.com/salaros
-* Aristarkh Zagorodnikov    https://github.com/onyxmaster
-* Samir Boulema             https://github.com/sboulema
-* Dmitry Lukyanov           https://github.com/DmitryLukyanov
-* Andrea Balducci           https://github.com/andreabalducci
-* Sergei Lipin              https://github.com/prchaoz
-
-If you have contributed and we have neglected to add you to this list please contact one of the maintainers to be added to the list (with apologies).
+Thank you [everyone](https://github.com/mongodb/mongo-csharp-driver/graphs/contributors) who has contributed to this project.
```
Comment (JamesKovacs) [inline README.md:79]: Thank you [everyone]( => Thank you to [everyone](

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Looks good. Minor correction.

**Review by rishitb-mongodb — APPROVED** [review]

**Pull request merged**